assets/css/base/base.css

```
@@ -69,6 +70,19 @@
   --focus-style-button-box-shadow: 0 0 0 6px var(--color-accent);
 }
 
+@supports(oklch(from orange)) {
+  @media (color-gamut: p3) {
```

Use correct `@supports` syntax and `dynamic-range: high` feature query:

```suggestion
@media (dynamic-range: high) {
  @supports(background: oklch(from orange l c h)) {
```